GUI/QtQuick/qml/WalletProperties/ChangePassword.qml

```diff
@@ -135,4 +168,13 @@ ColumnLayout {
         new_password.input_text = ""
         confirm_password.input_text = ""
     }
+
+    function checkPasswordLength()
+    {
+        if (new_password.input_text.length < 6) {
```

Comment:
Shouldn't you call `bsApp.validatePassword` here instead?

Reply:
I will follow your opinion